teacher.rb

```diff
@@ -0,0 +1,11 @@
+# teacher with there specialization and servies
+class Teacher < Person
+  def initialize(_specialization)
+    super()
+    @specialization = specialization
+  end
```

Comment:
- [ ] The other parameters for creating a person needs to be provided when creating a teacher. Kindly add them.

  ```
  def initialize(specialization, age, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @specialization = specialization
  end
  ```